crates/forge-runner/src/lib.rs

```diff
@@ -190,6 +191,12 @@ pub async fn run_tests_from_crate(
 
         print_test_result(&result);
 
+        if runner_config.save_trace_data {
+            if let AnyTestCaseSummary::Single(result) = &result {
+                save_trace_data(result);
```

Comment:
If user has only fuzzing tests this will fail silently

Reply:
Specification mentions it is not for fuzz tests